lib/bulk_upload/bulk_upload.rb

```diff
@@ -3,9 +3,11 @@
 module BulkUpload
   module BulkUpload
     def self.add_location_data(data, organisation)
-      data.each do |row|
-        location = organisation.locations.new(address: row[0], postcode: row[1])
-        row.slice(2..).each do |ip_address|
+      data_without_header = data.drop(1)
```

Comment:
It may be more obvious to remove the header in UploadForm. We don't do anything with the headers after reading them in anyway